README.md

```diff
@@ -93,7 +93,7 @@ Classics are common patterns that return in Round The Table matches, noticably b
 - The player saving the other will loose its life if the player makes a mistake. This way there is always a risk involved helping others.
 
 ### Promoting Diversity
-Diversity at HackerOne is a hot topic right now. We aim to be as diverse as possible. Not only when working on world-class product, but also during the teambuilding. Therefore any diverse playing member (women, foreigners, visitors from SF) will start any round with one life. That excludes players who own Frisian passport, and players from Brabant.
+Diversity at HackerOne is a hot topic. We aim to be as diverse as possible. Not only when working on a world-class product, but also during teambuilding. Therefore any diverse playing member (women, foreigners, visitors from SF) will start any round with one life. That excludes players who own Frisian passport and players from Brabant.
```

Comment:
So basically it's not about diversity anymore but about the level of the player. But then how do you decide what level the player has reached?